run

```diff
@@ -158,45 +158,42 @@ while [[ -n "${1:-}" ]] && [[ "${1:0:1}" == "-" ]]; do
             shift
         ;;
         -h|--help) echo "$USAGE"; exit ;;
-        -v|--version) echo "Generated from generator-canonical-webteam@3.4.1"; exit ;;
+        -v|--version) echo "Generated from generator-canonical-webteam@3.4.2"; exit ;;
         *) invalid "Option '${key}' not recognised." ;;
     esac
     shift
 done
 
-start_mongo () {
-    # Create isolated network
-    if ! docker network inspect ${network_name} &> /dev/null; then
-        docker network create ${network_name}
-    fi
+start_django_db () {
```

Comment:
Could this be `start_postgres` instead?

Reply:
Actually, [this might apply here too](https://github.com/canonical-web-and-design/360.canonical.com/pull/424#issuecomment-553098307).